src/git-town

```diff
@@ -2,6 +2,14 @@
 source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/helpers/helpers.sh" --bypass-automatic-configuration --bypass-environment-checks
 
 
+# Installs the autocompletion definition for Fish shell
+function install_fish_autocompletion {
+  run_command_outside_git "mkdir -p ~/.config/fish/completions"
+  run_command_outside_git "ln -s /usr/local/Cellar/git-town/0.5/libexec/autocomplete/git.fish ~/.config/fish/completions/git.fish"
```

Comment:
This depends on the user installing via homebrew, which is not guaranteed. Why not follow the instructions on the README?